make/migrations/postgresql/0050_2.2.0_schema.up.sql

```diff
@@ -455,3 +455,58 @@ BEGIN
     END LOOP;
 END $$;
 
+/*migrate robot_token_duration from minutes to days if exist*/
+DO $$
+DECLARE
+   properties_info record;
+   duration_in_minutes text;
+   duration_in_days integer;
+BEGIN
+  SELECT INTO properties_info * FROM properties WHERE k = 'robot_token_duration';
+  IF properties_info IS NOT NULL THEN
+    duration_in_minutes = properties_info.v;
+    duration_in_days = cast(duration_in_minutes as integer) / 60 / 24;
+    update properties set v = cast(duration_in_days as text)  WHERE k = 'robot_token_duration';
+  END IF;
+END $$;
+
+/*
+Common vulnerability reporting schema.
+Github proposal link : https://github.com/goharbor/community/pull/145
+*/
+
+-- --------------------------------------------------
+--  Table Structure for `main.VulnerabilityRecord`
+-- --------------------------------------------------
+CREATE TABLE IF NOT EXISTS "vulnerability_record" (
+    "id" serial NOT NULL PRIMARY KEY,
+    "cve_id" text NOT NULL DEFAULT '' ,
+    "registration_uuid" text NOT NULL DEFAULT '',
+    "digest" text NOT NULL DEFAULT '' ,
```

Comment:
This column is redundant. We have the artifact `digest` column in the `scan_report` table. We do not support layer id or layer digest in this release. Neither it's specified in the Pluggable Scanners API v1.1